docs/lineage/dagster.md

```diff
@@ -1,70 +1,187 @@
 # Dagster Integration
```

Comment:
One additional thing to add:

Which version of Dagster we tested this against.